content/docs/reference-events.md

```diff
@@ -31,15 +31,13 @@ number timeStamp
 string type
 ```
 
-> Note:
+> ध्यान दें:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> V0.14 के अनुसार, किसी इवेंट हैंडलर से `false` वापस करना अब event propagation को रोक नहीं पाएगा। इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को मैन्युअल रूप से, उपयुक्त के रूप में चालू/ट्रिगर किया जाना चाहिए।
 
-### Event Pooling {#event-pooling}
+### इवेंट इकट्ठा करना {#event-pooling}
 
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` को इकट्ठा किया जाता है। इसका मतलब यह है कि `SyntheticEvent` object का पुन: उपयोग किया जाएगा और ईवेंट कॉलबैक लागू होने के बाद सभी गुणों को null कर दिया जाएगा। यह निष्पादन कारणों से है। जैसे, आप ईवेंट को asynchronous तरीके से एक्सेस नहीं कर सकते.
```

Comment:
`निष्पादन` => `परफॉरमेंस`

Reply:
Can you remove line break at line 41.
I think line break was added with this fix

Reply:
> Can you remove line break at line 41.
> I think line break was added with this fix

ignore this. my bad. 
that was needed